### PR TITLE
Strip Social Urls to Decrease Their Length

### PR DESCRIPTION
**Pull request by sigmachirality**
Uses a regex to strip "https" and "www" from social urls.

Maybe we could format social urls in specific ways (for example, displaying twitter page links as @twitterusername) instead?

### CODE REVIEW

File: components/ClubPage/SocialLink.js [hunk 1]
```diff
@@ -1,14 +1,17 @@
+const stripUrl = url => {
+  return url.replace(/^(?:https?:\/\/)?(?:www\.)?/i, '')
```
Comment (cacabo) [inline components/ClubPage/SocialLink.js]: what about `http`?

Comment (cacabo) [reply components/ClubPage/SocialLink.js]: also you probably want to make sure that this is at the start of the url if there's an edge case where http is somewhere later like in a query param? might be better off not using a regex

Comment (ezwang) [reply components/ClubPage/SocialLink.js]: The question mark makes the `s` optional (matches both `http` and `https`). The `^` ensures that this regex only matches the start of the url.

Comment (sigmachirality) [reply components/ClubPage/SocialLink.js]: Screenshot from 2020-01-02 17-42-25

I'm no regex expert but it seems to work for me

**Comment by sigmachirality** [comment]
going to yeet

**Pull request merged**